### PR TITLE
test: add web worker tests for .ts and .tsx worker file

### PR DESCRIPTION
**Pull request by jerome-benoit**

### CODE REVIEW

File: test/js/web/workers/worker-fixture.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+postMessage("initial message");
```
Comment (paperclover) [inline test/js/web/workers/worker-fixture.tsx]: can you add actual jsx syntax into here. idk if jsx elements are transferrable due to the symbol, but they should be usable.

File: test/js/web/workers/worker-fixture.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+declare var self: Worker;
```
Comment (paperclover) [inline test/js/web/workers/worker-fixture.ts:1]: i guess `declare var` is enough to count as ts syntax. i imagined seeing some type annotation at all but this works.

**Review by paperclover — CHANGES_REQUESTED** [review]
looks good. one nitpick

File: test/js/web/workers/worker-fixture.ts (hunk 2 above)
Comment (jerome-benoit) [reply test/js/web/workers/worker-fixture.ts:1]: I can add some typing.

File: test/cli/run/run_command.test.ts [hunk 3]
```diff
@@ -5,16 +5,16 @@ import { bunEnv, bunExe } from "harness";
...
+      stdout: "pipe",
+      stderr: "pipe",
     });
-})
\ No newline at end of file
+    expect(stdout.toString()).toBeEmpty();
+    expect(stderr.toString()).toMatch(/Script not found/);
+    expect(exitCode).toBe(1);
+  });
+});
```
Comment (jerome-benoit) [inline test/cli/run/run_command.test.ts]: `bun fmt` formatting change.

**Review by paperclover — APPROVED** [review]

**Comment by paperclover** [comment]
running tests, though there are conflicts. sorry for long review time.

**Comment by jerome-benoit** [comment]
> running tests, though there are conflicts. sorry for long review time.

I will solve them ASAP, unless you can do it before I can.

**Review by paperclover — APPROVED** [review]

**Comment by jerome-benoit** [comment]
@paperdave: still interested to merge it?

**Pull request closed without merging**